relayer/ibc-client.go

```diff
@@ -0,0 +1,130 @@
+package relayer
+
+import (
+	"fmt"
+
+	clienttypes "github.com/cosmos/cosmos-sdk/x/ibc/core/02-client/types"
+	ibcexported "github.com/cosmos/cosmos-sdk/x/ibc/core/exported"
+	tmclient "github.com/cosmos/cosmos-sdk/x/ibc/light-clients/07-tendermint/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+	"golang.org/x/sync/errgroup"
+)
+
+// GetIBCCreateClientHeader updates the off chain tendermint light client and
+// returns an IBC Update Header which can be used to create an on-chain
+// light client.
+func (c *Chain) GetIBCCreateClientHeader(dst *Chain) (*tmclient.Header, error) {
```

Comment:
This is great, but I think we should abstract this to return a Header interface instead? Can be an improvement for a follow-up

Reply:
yes, let's do as a separate issue. No point in doing this until there are much larger relayer refactors

Reply:
#420